ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_stun_dtls_profile_name.schema.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
```

Comment:
May be one schema fragment for stun dtls named as "wan_stun_dtls" which can have both the keys: "wan_stun_dtls_disable" and "wan_stun_dtls_profile_name"?